sdk/tests/tutorials/ibor/test_properties.py

```diff
@@ -0,0 +1,137 @@
+        api_client = ApiClientBuilder().build(CredentialsSource.secrets_path())
+
+        cls.property_definitions_api = lusid.PropertyDefinitionsApi(api_client)
+        cls.transaction_portfolios_api = lusid.TransactionPortfoliosApi(api_client)
+        cls.instruments_api = lusid.InstrumentsApi(api_client)
+        cls.portfolios_api = lusid.PortfoliosApi(api_client)
+
+        instrument_loader = InstrumentLoader(cls.instruments_api)
+        cls.instrument_ids = instrument_loader.load_instruments()
+
+        cls.test_data_utilities = TestDataUtilities(cls.transaction_portfolios_api)
+
+    def get_guid(self):
+        # creates random alphanumeric code
+        return str(uuid.uuid4())[:12]
+
+    def test_create_portfolio_with_label_property(self):
+        # Details of property to be created
+        uuid = self.get_guid()
+        effective_date = datetime(year=2018, month=1, day=1, tzinfo=pytz.utc)
+
+        label_property_definition = models.CreatePropertyDefinitionRequest(
+            domain="Portfolio",
+            scope=TestDataUtilities.tutorials_scope,
+            code="fund-style-{}".format(uuid),
+            display_name="fund style",
+            life_time="Perpetual",
+            value_required=False,
+            data_type_id=models.resource_id.ResourceId(scope="system", code="string")
+
+        )
+
+        # create property definition
+        label_property_definition_request = self.property_definitions_api.create_property_definition(
+            label_property_definition)
+
+        # create property values
+        property_value = models.PropertyValue(label_value="Active")
+
+        # Details of new portfolio to be created
+        create_portfolio_request = models.CreateTransactionPortfolioRequest(
+            code="ud-{}".format(uuid),
+            display_name="portfolio-{}".format(uuid),
+            base_currency="GBP",
+            created=effective_date,
+            properties={
+                label_property_definition_request.key: models.PerpetualProperty(
+                    key=label_property_definition_request.key,
+                    value=property_value
+                )
+            }
+        )
+
+        # create portfolio
+        portfolio_request = self.transaction_portfolios_api.create_portfolio(scope=TestDataUtilities.tutorials_scope,
+                                                                             create_request=create_portfolio_request)
+
+        # get properties for assertions
+        portfolio_properties = self.portfolios_api.get_portfolio_properties(scope=TestDataUtilities.tutorials_scope,
+                                                                            code=portfolio_request.id.code).properties
+
+        label_property = portfolio_properties[label_property_definition_request.key]
+
+        # Perform assertions on keys, codes and values
+        assert (list(portfolio_properties.keys())[0] == label_property_definition_request.key)
```

Comment:
these should be `self.assertEqual()` rather than `assert`

`assertEqual` is the unit test assertions whereas `assert` is a language feature that used during debugging

Reply:
Changed